synapseutils/copy_functions.py

```diff
@@ -239,6 +239,7 @@ def _copy_cached_file_handles(cache: Cache, copiedFileHandles: dict) -> None:
 def changeFileMetaData(
     syn: synapseclient.Synapse,
     entity: typing.Union[str, Entity],
+    name: str = None,
```

Comment:
I am going to make 1 change here to put name at the end of the parameters being passed in - This is because if someone were using this function without explicit kwarguments then this would break their code.

Reply:
Good catch! Thanks @BryanFauble